src/function/table_functions/call_functions.cpp

```diff
@@ -272,6 +272,10 @@ std::unique_ptr<TableFuncBindData> ShowConnectionFunction::bindFunc(ClientContex
     TableFuncBindInput* input, Catalog* catalog, StorageManager* /*storageManager*/) {
     std::vector<std::string> returnColumnNames;
     std::vector<std::unique_ptr<LogicalType>> returnTypes;
+    // Special case here Due to any -> string, but lack implicit cast
```

Comment:
I would instead add implicit cast logic to `Binder::bindInQueryCall` and remove the special code path below.